rabbitmq/tests/metrics.py

```diff
@@ -44,6 +44,11 @@
     'rabbitmq.queue.messages.publish.rate',
 ]
 
+# Present from 3.6
+Q_METRICS_36 = [
+    'rabbitmq.queue.head_message_timestamp',
+]
```

Comment:
1. I think it's cleaner if you do something like:
```
if RABBITMQ_VERSION >= version.parse("3.6"):
    Q_METRICS.extend([
        'rabbitmq.queue.head_message_timestamp',
    ])
```

Same could be applied for `E_METRICS/E_METRICS_35`.

That way we don't need more complex if/else later in `test_integration_e2e.py`.

2. You can use `from packaging import version` for version comparison.

```
>>> from packaging import version
>>> version.parse("2.3.1") < version.parse("10.1.2")
```